src/main/java/mops/gruppen2/domain/model/group/GroupOptions.java

```diff
@@ -0,0 +1,36 @@
+package mops.gruppen2.domain.model.group;
+
+import lombok.Value;
+
+//TODO: doooooodododo
+@Value
+class GroupOptions {
+
+    // Gruppe
+    boolean showClearname;
+    boolean hasBody;
+    boolean isLeavable;
+    boolean hasLink;
+
+    String customLogo;
+    String customBackground;
+    String customTitle;
+
+    // Integrations
+    boolean hasMaterialIntegration;
+    boolean hasTermineIntegration;
+    boolean hasPortfolioIntegration;
+
+    static GroupOptions DEFAULT() {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [The static method name 'DEFAULT' doesn't match '[a-z][a-zA-Z0-9]*'](https://app.codacy.com/manual/tobi_2/gruppenfindung/pullRequest?prid=5356872)